content/applications/finance/fiscal_localizations/new_zealand.rst

```diff
@@ -0,0 +1,17 @@
+===========
+New Zealand
+===========
+
+.. _new-zealand/employment-hero:
+
+Employment Hero New Zealand Payroll
+===================================
+
+If your business is already up and running with `Employment Hero <https://employmenthero.com/>`_,
+you can use our connector as an alternative payroll solution.
```

Comment:
I'd rather link our employment hero doc here and remove the seealso section, wdyt?

```suggestion
If your business is already up and running with :doc:`Employment Hero <employment_hero>`, you can
use our connector as an alternative payroll solution.
```